OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/WebViewActivity.java

```diff
@@ -35,6 +35,15 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         setContentView(R.layout.activity_webview);
         super.setToolbarAsActionBar();
 
+        // Check if the activity was opened due to a broadcast for which we have an IntentFilter
+        // for this activity in AndroidManifest.xml
+        if (getIntent().getData() != null) {
```

Comment:
There should be a check of scheme here i.e. 'org.edx.mobile.innerlinks' before getting the relevant data.